docs/openapi.json

```diff
@@ -504,6 +504,20 @@
             "name": "workflow_id_or_name",
             "required": true,
             "type": "string"
+          },
+          {
+            "description": "Steps of a workflow.",
+            "in": "body",
+            "name": "steps",
+            "required": false,
+            "schema": {
+              "description": "List of steps names to get logs.",
+              "items": {
+                "description": "step name.",
```

Comment:
If we end with a dot, then let's make it a phrase, as it were, so starting with capital: "Step name."

Reply:
This change is missing.